Code/UnitTests/CoreTest/World/WorldTest.cpp

```diff
@@ -198,6 +198,62 @@ namespace
   EZ_END_DYNAMIC_REFLECTED_TYPE;
   EZ_IMPLEMENT_WORLD_MODULE(VelocityTestModule);
   // clang-format on
+
+  class SimulationTestModule : public ezWorldModule
+  {
+    EZ_ADD_DYNAMIC_REFLECTION(SimulationTestModule, ezWorldModule);
+    EZ_DECLARE_WORLD_MODULE();
+
+  public:
+    SimulationTestModule(ezWorld* pWorld)
+      : ezWorldModule(pWorld)
+    {
+    }
+
+    virtual void Initialize() override
+    {
+      {
+        auto desc = EZ_CREATE_MODULE_UPDATE_FUNCTION_DESC(SimulationTestModule::AlwaysUpdate, this);
+        RegisterUpdateFunction(desc);
+      }
+
+      {
+        auto desc = EZ_CREATE_MODULE_UPDATE_FUNCTION_DESC(SimulationTestModule::SimulationUpdate, this);
+        desc.m_bOnlyUpdateWhenSimulating = true;
+        RegisterUpdateFunction(desc);
+      }
+    }
+
+    void SimulationUpdate(const UpdateContext&)
+    {
+      m_nTimesSimulationUpdateCalled++;
+      m_bWasSimulationUpdateCalledLast = true;
+    }
+
+    void AlwaysUpdate(const UpdateContext&)
+    {
+      m_nTimesAlwaysUpdateCalled++;
+      if (m_bToggleSimulationNextAlwaysUpdate)
+      {
+        GetWorld()->SetWorldSimulationEnabled(!GetWorld()->GetWorldSimulationEnabled());
+        m_bToggleSimulationNextAlwaysUpdate = false;
+      }
+
+      m_bWasSimulationUpdateCalledLast = false;
+    }
+
+    ezInt32 m_nTimesSimulationUpdateCalled = 0;
```

Comment:
For integer we use the "i" prefix, so "m_iTimesSimulationUpdateCalled" etc. See https://github.com/ezEngine/ezEngine/blob/dev/CodingGuidelines.md

Reply:
Fixed in 63e08d6